builder/codegen/src/main/java/io/helidon/builder/codegen/CustomMethods.java

```diff
@@ -225,6 +232,46 @@ private static GeneratedMethod factoryMethod(Errors.Collector errors,
                                    codeGenerator);
     }
 
+    private static List<CustomConstant> findConstants(TypeInfo customMethodsType,
+                                                      Errors.Collector errors) {
+        return customMethodsType.elementInfo()
+                .stream()
+                .filter(ElementInfoPredicates::isField)
+                .filter(ElementInfoPredicates.hasAnnotation(PROTOTYPE_CONSTANT))
+                .map(it -> {
+                    if (!it.elementModifiers().contains(Modifier.STATIC)) {
+                        errors.fatal(it,
+                                     "A field annotated with @Prototype.Constant must be static, final, "
```

Comment:
Your message seems contradictory - it needs to be static ... "Not static." -> should be removed.

Reply:
It actually is correct - the field needs to be static. The "Not static." is the error message - your field is not static.